docker-compose.yml

```diff
@@ -35,6 +35,12 @@ services:
       AWS_SECRET_ACCESS_KEY: ${AWS_SECRET_ACCESS_KEY}
       HEADER_ACCEPT: ${HEADER_ACCEPT}
       HEADER_X_API_KEY: ${HEADER_X_API_KEY}
+      KAFKA_ENABLE_AUTO_COMMIT: ${KAFKA_ENABLE_AUTO_COMMIT}
+      KAFKA_AUTO_COMMIT_INTERVAL_MS: ${KAFKA_AUTO_COMMIT_INTERVAL_MS}
+      KAFKA_SESSION_TIMEOUT_MS: ${KAFKA_SESSION_TIMEOUT_MS}
+      GOOGLE_APPLICATION_CREDENTIALS: '/google/gcp_service_account.json'
```

Comment:
The same as above

Reply:
The value of the environment variable is mounted to /google/gcp_service_account.json and then that file is referenced in the application as GOOGLE_APPLICATION_CREDENTIALS.